### ISSUE TITLE
CVE-2021-3493 security VULNERABILITY

### ISSUE THREAD

**Issue by n8x0b7T** [pop-os/linux#51]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="20.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy
LOGO=distributor-logo-pop-os

Linux pop-os 5.8.0-7642-generic pop-os/pop#47~1614007149~20.10~82fb226-Ubuntu SMP Tue Feb 23 02:59:01 UTC  x86_64 x86_64 x86_64 GNU/Linux 
 

CVE-2021-3493 is exploitable on the latest version of pop os.
It is patched in Ubuntu

This vulnerability can easily be reproduced with the steps given in this article:
[https://ssd-disclosure.com/ssd-advisory-overlayfs-pe/](https://ssd-disclosure.com/ssd-advisory-overlayfs-pe/)

**Comment by jackpot51** [comment]
We are on 5.11 kernel now, why are you still on 5.8?

**Comment by n8x0b7T** [comment]
Should the new kernel be installed through updates in the popos store?

If not how to update my kernel?

**Comment by jackpot51** [comment]
Yes, it should be. You should have this version: ` 5.11.0-7614.15~1618626693~20.10~ecb25cd`

You can update with apt-get from a terminal as well:

```
sudo apt-get update
sudo apt-get upgrade
```

**Comment by stickr** [comment]
Some folks are stuck using 5.8 due to issue 45 https://github.com/pop-os/linux/issues/45 so I'd believe this issue will still affect them until 5.11 proves stable enough to upgrade to.

**Comment by daviddallet** [comment]
@jackpot51 5.8 I can confirm, 5.8 is something suggested by support for a lemp10 popos 20.04 in a case of instability with 5.11 that looks similar to #45.
